pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -220,6 +171,88 @@ def _write_packet_to_processor(self, body):
         self._processor_sock_wfile.write(body + '\n')
         self._processor_sock_wfile.flush()
 
+    def _log_and_encode_exception(self, formatted_exception, log_level='warn'):
+        getattr(self._logger, log_level)(formatted_exception)
+        return self._json_encoder.encode({
+            'body': formatted_exception,
+            'body_encoding': 'text',
+            'content_type': 'text/plain',
+            'status_code': 500,
+        })
+
+    def _resolve_event_message_length(self):
+
+        # used for the first message, to determine the body size
+        int_buf = bytearray(4)
+
+        should_be_four = self._processor_sock.recv_into(int_buf, 4)
+
+        # client disconnect
+        if should_be_four < 4:
```

Comment:
it can never be > 4, as you ask to read 4 bytes only. thought you _may_ read less, and in that case you will fail the process.
to make the code more readable, I will `<` => `!=`